### PR TITLE
Add show image from gallery and fix bug when delete user

### PR DESCRIPTION
**Pull request by VietHuy11**
My treatment of deleting user is still not effective

### CODE REVIEW

File: app/src/main/java/theintership/my/signin_signup/fragment/frag_signup_creating_account.kt [hunk 1]
```diff
@@ -135,58 +143,57 @@ class frag_signup_creating_account : Fragment(R.layout.frag_signup_creating_acco
...
+//                var delete_phone_email_account = false
+//                var user = Firebase.auth.currentUser
+//
+//                //Delete user auth
+//                user!!.delete().addOnCompleteListener(signup1activity) { task ->
+//                    if (task.isSuccessful) {
+//                        delete_user_auth = true
+//                        if (delete_user_auth && delete_phone_email_account && delete_user_info) {
+//                            move_to_frag_signing()
+//                        }
```
Comment (kics223w1) [inline app/src/main/java/theintership/my/signin_signup/fragment/frag_signup_creating_account.kt:160]: You forgot to delete these comment

File: app/src/main/java/theintership/my/signin_signup/fragment/frag_signup_creating_account.kt [hunk 2]
```diff
@@ -135,58 +143,57 @@ class frag_signup_creating_account : Fragment(R.layout.frag_signup_creating_acco
...
+//                    }
+//                }
+//
+//                //Deletet user_info
```
Comment (HUy2221) [inline app/src/main/java/theintership/my/signin_signup/fragment/frag_signup_creating_account.kt:164]: You forgot to delete these comment

**Review by kics223w1 — APPROVED** [review]
Ok let's merge

**Review by HUy2221 — APPROVED** [review]
Ok Let's go

**Pull request merged**